setup/so-whiptail

```diff
@@ -33,6 +33,24 @@ whiptail_airgap() {
 	INTERWEBS=$(echo "${INTERWEBS^^}" | tr -d ' ')
 }
 
+whiptail_analyst_install() {
+
+	[ -n "$TESTING" ] && return
+
+	read -r -d '' message <<- EOM
+	
+	Welcome to the Security Onion Analyst Workstation install!
+
+	You can choose to join this workstation to an existing grid?
+
+	NOTE: Selecting "no" will enable X Windows and set it to load at boot.
```

Comment:
I'm guessing that selecting either yes or no still enables the graphical desktop?

Reply:
Yes but this is only if you chose to do an analyst install from the ISO.